HyperPlay-Gaming/hyperplay-desktop-client
# [FIX] Back to previous position in library

FIX #544 

https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/assets/26871415/bca0ef19-8100-46de-b509-992c1bf98d4c

### How to test

- On the library, scroll down and click on any game card. 
- Click on the 'Back' Button
- Should get back to the previous position

Might not work on small libraries with just a few games.

---

Use the following Checklist if you have changed something on the Backend or Frontend:
